MM-18089 - Custom Admin Settings (#72)
mattermost/mattermost-plugin-demo: webapp/README.md

![custom-setting](docs/admin_setting_full_screen.png)

> - `SecretMessage`: Custom setting component that is registerd with the option to show title enabled and is displayed on the right column of the System Console setting page.

Reviewer: ```suggestion
- `SecretMessage`: Inline, custom setting component shown on the right column of the System Console setting page.
```
